docker/docker-static-build.sh

```diff
@@ -52,4 +52,5 @@ container_id="$("$virtualisation_engine" create alpine-tezos)"
 for b in "${binaries[@]}"; do
     "$virtualisation_engine" cp "$container_id:/tezos/$b" "$b"
 done
+printf "%s\n" "${binaries[@]}" > "/tmp/binaries.txt"
```

Comment:
It is not the correct place to get installable packages list. You can always get them by importing our python script, thus this does not require any additional file at all. See [here](https://github.com/serokell/tezos-packaging/blob/master/docker/build/ubuntu/build.py#L18).